Makefile

```diff
@@ -40,84 +40,57 @@ cri_version   =  release-1.26
 
-.PHONY: aer
-aer: proto ## Initialize and compile aer
-	@$(cargo) clippy -p aer
-	@$(cargo) install --path ./aer --debug --force
+.PHONY: test
+test: proto libs-test auraed-test auraescript-test aer-test ## Run tests (does not include ignored tests)
 
-musl: ## Add target for musl
-	rustup target add $(uname_m)-unknown-linux-musl
+.PHONY: test-all
+test-all: proto libs-test-all auraed-test-all auraescript-test-all aer-test-all ## Run tests (includes ignored tests)
 
-.PHONY: auraed
-auraed: musl proto ## Initialize and static-compile auraed with musl
-	@$(cargo) clippy --target $(uname_m)-unknown-linux-musl -p auraed
-	@$(cargo) install --target $(uname_m)-unknown-linux-musl --path ./auraed --debug --force
+.PHONY: install
+install: proto auraed-debug auraescript-debug aer-debug ## Build and install (debug) 🎉
 
-.PHONY: check-docs
-check-docs: # spell checking
-	@vale --no-wrap --glob='!docs/stdlib/v0/*' ./docs
+.PHONY: release
+release: lint test auraed-release auraescript-release aer-release ## Build and install (release) 🎉
 
 .PHONY: docs
-docs: proto crate stdlibdocs ## Assemble all the /docs for the website locally.
-	cp -rv README.md docs/index.md # Special copy for the main README
-	cp -rv api/README.md docs/stdlib/index.md # Special copy for the main README
-
-## Generate the docs for the stdlib from the .proto files
-ifeq (, $(wildcard /usr/local/bin/protoc-gen-doc))
-stdlibdocs:
-	$(error "No /usr/local/bin/protoc-gen-doc, install from https://github.com/pseudomuto/protoc-gen-doc")
-else
-stdlibdocs:
-	protoc --plugin=/usr/local/bin/protoc-gen-doc -I api/v0/discovery -I api/v0/observe -I api/v0/cells --doc_out=docs/stdlib/v0 --doc_opt=markdown,index.md:Ignore* api/v0/*/*.proto --experimental_allow_proto3_optional
-endif
+docs: proto docs-crates docs-stdlib docs-other docs-lint ## Assemble all the /docs for the website locally.
 
-crate: musl ## Build the crate (documentation)
-	$(cargo) doc --target $(uname_m)-unknown-linux-musl --no-deps --package auraed
-	$(cargo) doc --no-deps --package auraescript
-	$(cargo) doc --no-deps --package aurae-client
-	$(cargo) doc --no-deps --package aer
-	cp -rv target/doc/* docs/crate
+.PHONY: prcheck
+prcheck: install lint test-all docs ## Meant to mimic the GHA checks (includes ignored tests)
 
-serve: docs ## Run the aurae.io static website locally
-	sudo -E ./hack/serve.sh
+#------------------------------------------------------------------------------#
 
-test: musl proto ## Run the tests
-	@$(cargo) test --target $(uname_m)-unknown-linux-musl --workspace --exclude auraescript
-	@$(cargo) test -p auraescript
+# Setup Commands
 
-test-all: musl proto ## Run the tests (including ignored)
-	@sudo -E $(cargo) test --target $(uname_m)-unknown-linux-musl --workspace --exclude auraescript -- --include-ignored
-	@$(cargo) test -p auraescript -- --include-ignored
+.PHONY: pki
+pki: certs ## Alias for certs
+.PHONY: certs
+certs: clean-certs ## Generate x509 mTLS certs in /pki directory
```

Comment:
do we need to clean before creation?  i'm always a bit suspicious of targets that auto-clean.

Reply:
I assume `certs` will fail (the clean step was there before) if the pki directory is not empty. I don't know what the correct approach should be and don't have an opinion. Your call.